README.md

```
@@ -20,18 +20,32 @@ To help you get started with Nakazawa, follow these simple steps:
    cd nakazawa
    ```
 
-2. **Open the Application**:
-   Open the `index.html` file in your preferred web browser:
+2. **Install Dependencies**:
+   Install the required dependencies using npm:
    ```bash
```

Markdownlint (0.37.0)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)